content/src/view.js

```diff
@@ -3176,6 +3281,165 @@ function setupHpanelBox(box) {
   });
 }
 
+function arrow(pane, arrow_list, traceEventNum, show_fade) {
```

Comment:
What exactly does this function draw? a single arrow? all the arrows? some of the arrows? (is there only one arrow drawn at a time, ever? if so, why the arrow_list? can you just pass the single arrow?)